.circleci/config.yml

```
@@ -0,0 +1,65 @@
+var_1: &docker_image circleci/node:10.13.0-browsers
+var_2: &cache_key raiden-webui-{{ .Branch }}-{{ checksum "package-lock.json" }}
+
+
+anchor_1: &defaults
+  working_directory: ~/src
+  docker:
+    - image: *docker_image
+
+anchor_2: &root_package_lock_key
+  key: *cache_key
+
+anchor_3: &attach_options
+  at: .
+
+
+version: 2
+jobs:
+  install:
+    <<: *defaults
```

here you can choose the executor by the native key

```suggestion
executor:
  name: base-executor
```

later on you can even pass an executor through parameters from the workflow down to the job. This is useful if you would like to test on different node versions for example. Its also type secure (you have to declare type when defining a parameter)